library/blivet.py

```diff
@@ -1469,6 +1469,8 @@ def run_module():
                                                           fs_type=dict(type='str'),
                                                           mount_point=dict(type='str'),
                                                           name=dict(type='str'),
+                                                          raid_disks=dict(type='list', elements='str', default=list()),
+                                                          raid_level=dict(type='str'),
```

Comment:
This should be done in a separate commit - and I'm quite puzzled as how https://github.com/linux-system-roles/storage/pull/241 could have passed without this - this should have been part of https://github.com/linux-system-roles/storage/pull/241

Reply:
Do you mean a separate PR? Tests for #241 passed because it wasn't rebased.

Reply:
> Do you mean a separate PR?

Yes - because it is unrelated to this PR

> Tests for #241 passed because it wasn't rebased.

ah I see - https://github.com/linux-system-roles/storage/pull/223 was merged too - so what should have happened is when https://github.com/linux-system-roles/storage/pull/223 was merged, all of the existing PRs should also have been rebased and fixed

Reply:
Ok, I moved the commit to a new PR #246

Reply:
> Ok, I moved the commit to a new PR #246

Thanks!  merged